src/cloudant/result.py

```diff
@@ -359,6 +359,17 @@ def _parse_data(self, data):
         """
         return data.get('rows', [])
 
+    def all(self):
+        """
+        Retrieve all results.
+
+        Specifying a ``limit`` parameter in the ``Result`` constructor will
```

Comment:
The `page_size` parameter is not respected when accessing results via slice. This is already noted in the docs here:
>:param int page_size: Sets the page size for result _iteration_.